### PR TITLE
Add script to print certificate path

### PR DESCRIPTION
**Pull request by wenwenchenbosch**
so that it is easier for user to find certificates if just use pip packages

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -1,2 +1,3 @@
 recursive-include kuksa_certificates *
 recursive-include doc *
+recursive-include kuksa_viss_client *
```
Comment (wenwenchenbosch) [inline MANIFEST.in:3]:
```suggestion
recursive-include kuksa_viss_client *2
```

**Comment by SebastianSchildt** [comment]
Did we (intentionally) loose the capability to start a development version directly?

```
sebastian@sebastian-VirtualBox:~/kuksa.val/kuksa_viss_client$ python3 __main__.py 
Traceback (most recent call last):
  File "__main__.py", line 25, in <module>
    from kuksa_certificates import __certificate_dir__
ImportError: cannot import name '__certificate_dir__' from 'kuksa_certificates' (/home/sebastian/.local/lib/python3.8/site-packages/kuksa_certificates/__init__.py)
```

**Comment by wenwenchenbosch** [comment]
@SebastianSchildt the issue should be fixed now

**Comment by wenwenchenbosch** [comment]
also solve #164

**Review by SebastianSchildt — CHANGES_REQUESTED** [review]
One question: Didn't we decide, the client should also print the default cert dir when starting?

But I still have another problem. I can now start the client locally (from github), but the new command crashes

```
 python3 __main__.py 
Welcome to kuksa viss client 0.1.7.post126

                  `-:+o/shhhs+:`                  
                ./oo/+o/``.-:ohhs-                
              `/o+-  /o/  `..  :yho`              
              +o/    /o/  oho    ohy`             
             :o+     /o/`+hh.     sh+             
             +o:     /oo+o+`      /hy             
             +o:     /o+/oo-      +hs             
             .oo`    oho `oo-    .hh:             
              :oo.   oho  -+:   -hh/              
               .+o+-`oho     `:shy-               
                 ./o/ohy//+oyhho-                 
                    `-/+oo+/:.             


connect to wss://127.0.0.1:8090
Websocket connected!!
Test Client> info 
Kuksa viss client 0.1.7.post126
Uri: https://github.com/eclipse/kuksa.val
Author: Sebastian Schildt, Naresh Nayak, Wenwen Chen
Copyright: Copyright 2020 Robert Bosch GmbH
EXCEPTION of type 'AttributeError' occurred with message: 'module 'kuksa_certificates' has no attribute '__certificate_dir__''
To enable full traceback, run the following command: 'set debug true'
Test Client> 
```

**Review by SebastianSchildt — APPROVED** [review]
Now I finally can't break it :)

**Pull request merged**